frontend/packages/dev-console/src/utils/pipeline-augment.ts

```diff
@@ -182,12 +182,12 @@ export interface Param {
 }
 
 export interface PipelineParam extends Param {
-  default?: string;
+  default?: string | string[];
   description?: string;
 }
 
 export interface PipelineRunParam extends Param {
-  value: string;
+  value: string | string[];
```

Comment:
Apparently I failed to change the type in my Array PR: https://github.com/openshift/console/pull/4735